pkg/firehose/firehose.go

```diff
@@ -213,13 +209,13 @@ type FirehoseEvent struct {
 }
```

Comment:
Can you help me understand the rationale for building the `FirehoseEvent` megastruct?